DuckDuckGo/AppDelegate.swift

```diff
@@ -1089,3 +1096,24 @@ private extension Error {
     }
 
 }
+
+final class LaunchOptionsHandler {
```

Comment:
Temporarily left this file in the AppDelegate.

Theoretically we could make an extension on `ProcessInfo` e.g.

```
extension ProcessInfo {
    var isUITesting: Bool {
        arguments.contains(Self.isUITesting)
    }
}
```

Because we need to read the value of the env variable from the user defaults we cannot mock the UserDefaults in extensions if we want to unit test this behaviour.